scripts/cleanup.sh

```diff
@@ -203,6 +204,33 @@ function delete_keys() {
   RUN_IBMCLOUD pi key-delete "${INFRA_ID}-key"
 }
 
+function delete_vpcs() {
+  echo "Deleting VPC and VPC Subnet"
+  RUN_IBMCLOUD is vpcs --output JSON
+  vpc_id="$(echo "${CMD_OUT}" | jq -r ".[]|select(.name==\"powervs-ipi\").crn")"
+  RUN_IBMCLOUD is subnet-delete -f "vpc-subnet-${INFRA_ID}" --vpc "${vpc_id}"
+  sleep 2
+  RUN_IBMCLOUD is vpc-delete -f "vpc-${INFRA_ID}"
+}
+
+function delete_dhcp() {
+    dhcp_id=$(cat test/terraform.cluster.tfstate | jq -r ".resources[]|select(.name==\"dhcp_service\")|select(.module==\"module.pi_network\").instances[0].attributes.dhcp_id")
```

Comment:
There is high chance that this will fail if not run from a proper directory, wondering if we can get the dhcp servers available in any instance and delete them

Reply:
oh, yes i'd missed this hard-coding.

as far as query-ing, yes we can do that too. i just didnt want to go that route b/c we can technically have many dhcp services in one service instance as long as only one is attached to a cloud connection. so i didn't (for our dev purposes) want to make any assumptions like that, and this seemed pretty straight-forward

Reply:
I don't want to debate a lot on the approach here because its not really worth because our official destroy code will take over this cleanup script soon :)

Reply:
right. just explaining. hope we can go ahead since this is hopefully very temporary